subsys/logging/log_backend_spinel.c

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log_backend.h>
+#include <logging/log_output.h>
+#include <openthread/platform/logging.h>
+#include <openthread/platform/uart.h>
+#include <platform-zephyr.h>
+#include "log_backend_std.h"
+
+#ifndef CONFIG_LOG_BACKEND_SPINEL_BUFFER_SIZE
+#define CONFIG_LOG_BACKEND_SPINEL_BUFFER_SIZE 0
+#endif
+
+static uint8_t char_buf[CONFIG_LOG_BACKEND_SPINEL_BUFFER_SIZE];
+static bool panic_mode;
+
+static int write(uint8_t *data, size_t length, void *ctx);
+
+LOG_OUTPUT_DEFINE(log_output, write,
+		  char_buf, sizeof(char_buf));
```

Comment:
nit: Do we need to break the line here, I guess it should all fit into a single line?

Reply:
Done